airbyte-integrations/connectors/source-youtube-data/manifest.yaml

```diff
@@ -111,12 +109,6 @@ definitions:
             type: RequestOption
             inject_into: request_parameter
             field_name: id
-      transformations:
-        - type: AddFields
-          fields:
-            - path:
-                - id
-              value: "{{ stream_partition.channel_id }}"
```

Comment:
I assume that it's removed due to "id" field is now present in the record due to removing snippet field from the record selector path. So we are good wit this change, but want to confirm since this is a primary key for the stream.